src/time-map.scm

```diff
@@ -113,12 +126,32 @@
 		(angle ax ay az bx by bz)
 	)
 )
+
+(define (angle_bw_sf xyz-a xx yy zz)
+	(let* ((sta (string-split xyz-a #\ ))
+		(ax (string->number(car sta)))
+		(ay (string->number(cadr sta)))
+		(az (string->number(caddr sta))))
```

Comment:
once again, we seem to have a cascade of string-to-number and number-to-string conversions, for the same values, over and over.   Please make up your mind as to which you want to use. please stop converting all the time.

You added a number API to the core atomspace bindings, despite my rather strongly recommending that it was a bad idea, and would only lead to chaos.   Well, the above three lines of code are a proof of that chaos.   You've got to pick an API, and stick to it.